.github/workflows/publish-dts-sdk.yml

```
@@ -0,0 +1,45 @@
+name: Publish Durable Task Scheduler to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*'  # Only run for tags starting with "v"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Extract version from tag
+        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV  # Extract version from the tag
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"  # Adjust Python version as needed
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build twine
+
+      - name: Build package from directory durabletask-azuremanaged
```

```suggestion
      - name: Build package from directory durabletask-azuremanaged
        working-directory: durabletask-azuremanaged
```

This block should look like:

```
      - name: Build package from directory durabletask-azuremanaged
        working-directory: ./durabletask-azuremanaged
        run: |
          python -m build
```

We don't need to `cd` which can get confusing. Let's just set the working directory (which is relative to the root)